addons/crm_project_issue/wizard/crm_lead2projectissue_wizard.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+
+
+class CrmLeadToProjectIssueWizard(models.TransientModel):
+    """ wizard to convert a Lead into a Project Issue and move the Mail Thread """
+    _name = "crm.lead2projectissue.wizard"
+    _inherit = 'crm.partner.binding'
+
+    project_id = fields.Many2one('project.project', string='Project', domain=[('use_issues', '=', True)])
+
+    @api.multi
+    def action_lead_to_project_issue(self):
+        # get the models
+        Issue = self.env['project.issue']
```

Comment:
Remove this object reference variable and check below comment.
![look-below](https://cloud.githubusercontent.com/assets/14940179/12080195/a44f6160-b210-11e5-877f-2af1870d24ca.jpg)